website/content/docs/auth/saml.mdx

```diff
@@ -175,6 +175,18 @@ Whether it should be an exact match or interpret `*` as a wildcard can be
 controlled by the [`bound_subjects_type`](/vault/api-docs/auth/saml#bound_subjects_type) and
 [`bound_attributes_type`](/vault/api-docs/auth/saml#bound_attributes_type) parameters.
 
+### Bound attributes for the Microsoft identity platform
+
+The bound attributes for the Microsoft identity platform requires
+`http://schemas.microsoft.com/ws/2008/06/identity/claims/groups` as the
+attribute name along with your group membership values. For example, a role with
```

Comment:
The example is great. I think if it were in its own paragraph it would stand out more.